### DISCUSSION TITLE
minio containers still starts up slowly after set fsGroupChangePolicy as "OnRootMismatch"

### DISCUSSION THREAD

**Discussion by YorckZhou** [minio/minio#20322] [Q&A] [answered]
I have a minio cluster runs on K8s. There are 4 replicas in the cluster, with 8 drives. There is only one bucket, with 47GB data, 298,123 objects.
I have set the minio config yaml as below
```
securityContext:
  fsGroup: 1001
  fsGroupChangePolicy: "OnRootMismatch"
```
as per below doc:
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods

It takes about one hour for a replica to start up before the change. 
After apply the above change, it takes about 24 to 45 minutes for a replica to start up. Most of the above time is consumed at the step of volume-permissions checking.

Please advise how to reduce the volume-permissions checking further. Thanks!

**Comment by harshavardhana** [comment, marked as answer]
This is nothing to do with MinIO, this is to do with k8s you need to use the right k8s version for this to avoid the changes.
